### ISSUE TITLE
Restic build test failure

### ISSUE THREAD

**Issue by copart** [restic/restic#1886]
Summary:
Trying to create an official rpm package on Fedora.  So far I have a restic package available on COPR. However, Fedora requires that all vendor dependencies are removed and use the packaged version.  This makes the package more difficult to get compiled.  At this time it looks like it compiles, however, it  failing a test.  I was wondering if anyone has a guess on what is causing the test to fail.

## Building Resting
It builds.  However, fail seen during TEST:

```
Testing: "/builddir/build/BUILD/restic-0.9.1/_build/src/github.com/restic/restic/internal/restic"
+ GOPATH=/builddir/build/BUILD/restic-0.9.1/_build:/usr/share/gocode
+ go test -buildmode pie -compiler gc -ldflags '-extldflags '\''-Wl,-z,relro   -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '\'''
# github.com/restic/restic/internal/restic
./node_unix_test.go:115: T.Skip call has possible formatting directive %T
FAIL    github.com/restic/restic/internal/restic [build failed]
```

## Steps to reproduce the behavior
mock build on Fedora Rawhide.  
spec: https://copart.fedorapeople.org/restic/restic.spec
srpm: https://copart.fedorapeople.org/restic/restic-0.9.1-2.fc29.src.rpm
`mock SRPMS/restic-0.9.1-2.fc29.src.rpm`

## Did restic help you or made you happy in any way?
I use it so much I want others to be able to install it in a easier way.  :)

**Comment by fd0** [comment]
Hey, thanks a lot for packaging restic! It's the same with Debian, which also requires properly packaging all upstream dependencies. The issue is that during testing, recent versions of the Go compiler treat calls to `Skip()` with a string which contains formatting characters (`%T` here) as an error, the call should be changed to `Skipf()`. I've fixed this yesterday in master here: abb18a830c7601b4ce873052031d0516217f74a6

Can you cherry-pick this patch?

**Comment by copart** [comment]
Adding your patch to the rpm did the trick!  Thank you!
